text/XXXX-element-pointer.md

```diff
@@ -0,0 +1,33 @@
+- Start Date: 2017-03-28
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Element Pointer
+
+## Summary
+
+Element Pointer is currently a special case and the only type that does not
+extend from an element which means it has special rules when it comes to
+serialisation and de-serialisations.
+
+## Motivation
+
+To aid simplicity and to remove complexity of Refract serialisation, we should
+move Element Pointer to be an element itself. This would be more consistent
+with other elements such as the Link element.
+
+## Detailed design
+
+The Element Pointer should become an Element Pointer element which can be
+described as follows:
+
+### Element Pointer
+
++ element: elementPointer (fixed)
++ attributes
+    + path (enum) - Path of the referenced element to transclude instad of element itself
+        + element (default)
```

Comment:
Just to note, we reserve it for object properties, but not sure about values (here it would be the value of the `path` property). With that said, it may be helpful to not use `element` as a possible value.

Second, do you want to require this value @kylef? I know you have expressed thinking about the Zen of Python, and it would remove a rule for the implementation (i.e. if `path` isn't set, do X, if not, do Y). It doesn't matter to me, so I'm just bringing it up if we want to start thinking about being more explicit with the serialization/deserialization.